Ginger/GingerPluginCore/ActionsLib/IRecord.cs

```diff
@@ -19,9 +19,17 @@
 
 namespace Amdocs.Ginger.Plugin.Core
```

Comment:
need to move "IRecord" under "RecordingLib" folder

Reply:
Done